Python/testing.ipynb

```diff
@@ -28,7 +28,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 107,
+   "execution_count": 5,
```

Comment:
If you're including this file as an example usage, I'd recommend that you reset, and rerun the entire notebook. Restart the kernel and use the "run all cells" (or something like that) command, to ensure that everything actually still works after modifying all of you code.